indra/newview/llreflectionmap.cpp

```diff
@@ -248,32 +248,28 @@ bool LLReflectionMap::getIsDynamic()
 
 bool LLReflectionMap::getBox(LLMatrix4& box)
 {
-    if (mViewerObject)
+    if (mViewerObject && !mViewerObject->isDead() &&
+        mViewerObject->mDrawable && !mViewerObject->mDrawable->isDead() &&
+        mViewerObject->getVolume() && mViewerObject->getReflectionProbeIsBox())
```

Comment:
This is likely fixed already. Note that mViewerObject::getReflectionProbeIsBox() doesn't exist on main.

Code was:
```
    if (mViewerObject)
    {
        LLVolume* volume = mViewerObject->getVolume();
        if (volume)
        {
            LLVOVolume* vobjp = (LLVOVolume*)mViewerObject;

            if (vobjp->getReflectionProbeIsBox())
            {
```

And crash was happening because mViewerObject wasn't LLVOVolume.

Reply:
viewer#1692 Crash at LLVOVolume::getReflectionProbeIsBox() https://github.com/secondlife/viewer/commit/51f5e09bd6457c8f90c1f63d24503e5bd8a51dab